### ISSUE TITLE
KeyboardProvider breaks translucent system navigation bar

### ISSUE THREAD

**Issue by whalemare** [kirillzyusko/react-native-keyboard-controller#119]
**Describe the bug**
When I wrap my app with 

```
return <KeyboardProvider statusBarTranslucent>{children}</KeyboardProvider>
```

Android system navigation bar becomes colored, not translucent

**Code snippet**
```
return <KeyboardProvider statusBarTranslucent>{children}</KeyboardProvider>
```

styles.xml
```xml
    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
        <item name="android:forceDarkAllowed" tools:ignore="NewApi">false</item>

        <item name="android:windowTranslucentStatus">true</item>
        <item name="android:windowTranslucentNavigation">true</item>
        <item name="android:statusBarColor">@android:color/transparent</item>
        <item name="android:navigationBarColor">@android:color/transparent</item>
    </style>
```

**Repo for reproducing**
*I can do it later if you need it*

**To Reproduce**
1. Wrap app with KeyboardProvider
2. Use translucent navigation
3. Run app

**Expected behavior**
<img width="285" alt="Снимок экрана 2023-02-23 в 12 56 00" src="https://user-images.githubusercontent.com/9128731/220874510-65bf998c-9ee7-405e-96d0-01d0be654fa3.png">

**Actual behavior**
<img width="294" alt="Снимок экрана 2023-02-23 в 12 54 46" src="https://user-images.githubusercontent.com/9128731/220874228-31675d00-482f-4029-b4bc-5c717cf1fe8b.png">

**Smartphone (please complete the following information):**
 - Desktop OS: MacOS
 - Device: Android Nexus 5 
 - OS: API 33
 - RN version: 0.71.2
 - RN architecture: old, paper
 - JS engine: Hermes
 - Library version: 1.4.3

**Additional context**
no applicable

**Comment by kirillzyusko** [comment]
Hi @whalemare 

Nice point! Actually I think it kind of expected behavior, since in the code I specify it like 

https://github.com/kirillzyusko/react-native-keyboard-controller/blob/9986f5f8950ba4473ca1a5963ac915997fe8d71c/android/src/main/java/com/reactnativekeyboardcontroller/managers/KeyboardControllerViewManagerImpl.kt#L47

Which literally means "apply default padding". I think I need to introduce `navigationBarTranslucent` prop to make it configurable. For now you can use `patch-package` and instead of this line simply specify `0` (just remove `insets?.getInsets(WindowInsetsCompat.Type.navigationBars())?.bottom ?:` and leave `0`).

Please, let me know, if the "fix" with substituting `0` instead of ternary operator actually helps or not. If it helps then I will create a PR soon 👀

**Comment by whalemare** [comment]
Yes, it helps! Thanks a lot

**Comment by kirillzyusko** [comment]
I've merged a PR that adds new property called `navigationBarTranslucent` on `KeyboardProvider`

So you'll need to use:

```tsx
<KeyboardProvider navigationBarTranslucent>
...
</KeyboardProvider>
```

In order not to break translucency specified by android styles.

**Comment by kirillzyusko** [comment]
Fixed in `1.4.4` 😊
